.eslintrc

```diff
@@ -40,6 +40,14 @@
   },
   "globals": {
     "Simulate": true,
-    "_satellite": true
+    "_satellite": true,
+    "shared-testing-elements": true
+  },
+  "settings": {
```

Comment:
this allows webpack to be the source of truth for our new alias of `test-helpers`